src/repos/UserRepo.ts

```diff
@@ -58,8 +60,13 @@ const getUserByUUID = async (uuid: string): Promise<User> => {
 /**
  * Return all users who follow a publication.
  */
-const getUsersFollowingPublication = async (pubID: PublicationID): Promise<User[]> => {
-  const users = await UserModel.find({ followedPublications: pubID });
+const getUsersFollowingPublication = async (pubSlug: string): Promise<User[]> => {
+  const publication = await PublicationRepo.getPublicationBySlug(pubSlug);
+  // WARNING TODO: linear scan on DB, inefficient <-- turn into a query that just gets users following publications
+  const users = await UserModel.find();
+  users.filter((u) => {
+    u.followedPublications.map((id) => id.id).includes(publication.id);
```

Comment:
you're iterating through with the map and then again with the includes, more efficient would be `u.followedPublications.some((id) => id.id === publication.id);` right because then you don't loop through twice?